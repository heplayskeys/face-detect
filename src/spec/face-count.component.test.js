import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import FaceCount from '../components/face-count/face-count.component';

describe('<FaceCount />', () => {
	it('returns correct face count snapshot with no faces', () => {
		const wrapper = shallow(<FaceCount />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});

	it('returns correct face count snapshot with faces', () => {
		const numFaces = 5;
		const wrapper = shallow(<FaceCount numFaces={numFaces} />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
});
