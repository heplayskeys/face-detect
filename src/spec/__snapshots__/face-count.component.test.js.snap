// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FaceCount /> returns correct face count snapshot with faces 1`] = `
<div
  className="mv2"
>
  <h3>
    I am able to grab 
    <span
      className="f2-ns f3 white"
    >
      5
    </span>
     faces from your image!
  </h3>
</div>
`;

exports[`<FaceCount /> returns correct face count snapshot with no faces 1`] = `
<p
  className="f4 mt3 mb3"
>
  I am unable to grab faces from your image.
</p>
`;
